src/NServiceBus.Core.Tests/Recoverability/SecondLevelRetries/DefaultRetryPolicyTests.cs

```diff
@@ -36,7 +36,7 @@ public void ShouldCapTheRetryMaxTimeTo24Hours()
             var moreThanADayAgo = now.AddHours(-24).AddTicks(-1);
             var headers = new Dictionary<string, string>
             {
-                {SecondLevelRetriesBehavior.RetriesTimestamp, DateTimeExtensions.ToWireFormattedString(moreThanADayAgo)}
+                {Headers.RetriesTimestamp, DateTimeExtensions.ToWireFormattedString(moreThanADayAgo)}
```

Comment:
@danielmarbach It is as correct as it was before the change, I think. What can be incorrect about spacing here?